### ISSUE TITLE
Run code with example corpus with error

### ISSUE THREAD

**Issue by svjack** [liao961120/concordancer#1]
I try the example corpus server.
and have below error:
<img width="646" alt="截屏2021-07-09 下午7 31 54" src="https://user-images.githubusercontent.com/27874014/125071925-826cbc80-e0ec-11eb-9129-b85af18acc2f.png">

**Comment by liao961120** [comment]
A query interface should pop up in your web browser.
Could you provide more context, e.g., the code you used to start the server?

**Comment by svjack** [comment]
![IMG_1056](https://user-images.githubusercontent.com/27874014/125146565-f93f9f00-e158-11eb-9b22-1bef7cffbc7e.jpeg)
i also curl to /query , it has status code 200. 
Can you provide me fully curl or request format to test ?

**Comment by svjack** [comment]
Where is your entry to retrieve “title” or the render source ?

**Comment by svjack** [comment]
i run your project in macos and i think this may be a front end problem

**Comment by liao961120** [comment]
I publish the front-end query interface at <https://concordancer.netlify.app>.

So after you run:
```python
server.run(C)
```

you can visit <https://concordancer.netlify.app> to query the corpus.

Also, if you are interested in sending requests by yourself, you can take a look at the source code in [`server.py`](https://github.com/liao961120/concordancer/blob/main/concordancer/server.py). The tedious part is escaping various special characters in CQL.

**Comment by svjack** [comment]
> I publish the front-end query interface at https://concordancer.netlify.app.
> 
> So after you run:
> 
> ```python
> server.run(C)
> ```
> 
> you can visit https://concordancer.netlify.app to query the corpus.
> 
> Also, if you are interested in sending requests by yourself, you can take a look at the source code in [`server.py`](https://github.com/liao961120/concordancer/blob/main/concordancer/server.py). The tedious part is escaping various special characters in CQL.

Where is the entry of https://concordancer.netlify.app/ in the server.py?
Or how can i init without 404 in local ?

**Comment by svjack** [comment]
![IMG_1057](https://user-images.githubusercontent.com/27874014/125148437-cbf8ee00-e164-11eb-864f-e4aeb3d5bb24.jpeg)
in your web site https://concordancer.netlify.app/

**Comment by liao961120** [comment]
Strange...
It does work on my computer
![image](https://user-images.githubusercontent.com/32927002/125148622-5ee65800-e166-11eb-8b53-ef864985ab15.png)

**Comment by svjack** [comment]
> Strange...
> It does work on my computer
> ![image](https://user-images.githubusercontent.com/32927002/125148622-5ee65800-e166-11eb-8b53-ef864985ab15.png)

I now run the code locally. And have try to install your package in a clean ubuntu.
Also can not render in the browser. 
Should i install vue? if i should , can you give me a guide of install it ?

**Comment by liao961120** [comment]
No, vue is not required. Did the browser show up `invalid cql syntax`? If it does, it means the server is responding, so it might be backend problems.

**Comment by svjack** [comment]
> No, vue is not required. Did the browser show up `invalid cql syntax`? If it does, it means the server is responding, so it might be backend problems.

Nope,
in ubuntu it is also 404 as macos

**Comment by liao961120** [comment]
You can test the URL below directly (with the browser or curl) after starting the server and see if any JSON is returned.
<http://localhost:1420/query?query=%22%E6%88%91%22&left=10&right=10>

**Comment by svjack** [comment]
> You can test the URL below directly (with the browser or curl) after starting the server and see if any JSON is returned.
> http://localhost:1420/query?query=%22%E6%88%91%22&left=10&right=10

This have response, but request to http://localhost:1420 directly, don't have response (404)

**Comment by liao961120** [comment]
That's working as expected, since the server does not listen on `/`. Only GET requests sent to  `/query` and `/export` will respond.

It seems that the failure is coming from the browser. What browser do you use? Chrome works fine on my computer.

**Comment by svjack** [comment]
> That's working as expected, since the server does not listen on `/`. Only GET requests sent to `/query` and `/export` will respond.
> 
> It seems that the failure is coming from the browser. What browser do you use? Chrome works fine on my computer.

macos i use safari 
ubuntu i use firefox

**Comment by svjack** [comment]
I macos i use chorme have pollowing result:
<img width="1301" alt="截屏2021-07-11 上午2 18 07" src="https://user-images.githubusercontent.com/27874014/125172829-6e10e880-e1ee-11eb-9abe-bc0cb8bb1915.png">

**Comment by liao961120** [comment]
It seems fine. <http://localhost:1420/> is not expected to response.

> ![IMG_1057](https://user-images.githubusercontent.com/27874014/125148437-cbf8ee00-e164-11eb-864f-e4aeb3d5bb24.jpeg)
> in your web site https://concordancer.netlify.app/

The figure above means that your browser is working fine (since `INVALID CQL SYNTAX` is a response from the server). Basically, the font-end just sends ajax GET request to <http://localhost:1420/query>:

![image](https://user-images.githubusercontent.com/32927002/125180076-79810580-e228-11eb-8ff1-efd899bc077a.png)

**Comment by svjack** [comment]
> It seems fine. http://localhost:1420/ is not expected to response.
> 
> > ![IMG_1057](https://user-images.githubusercontent.com/27874014/125148437-cbf8ee00-e164-11eb-864f-e4aeb3d5bb24.jpeg)
> > in your web site https://concordancer.netlify.app/
> 
> The figure above means that your browser is working fine (since `INVALID CQL SYNTAX` is a response from the server). Basically, the font-end just sends ajax GET request to http://localhost:1420/query:
> 
> ![image](https://user-images.githubusercontent.com/32927002/125180076-79810580-e228-11eb-8ff1-efd899bc077a.png)

I find the my import code yield warning about :
python`logging.warning("Skip download query interface")`
Dose this related with the error above ?

**Comment by liao961120** [comment]
Nope, as long as you use the query interface at <https://concordancer.netlify.app/>, you can ignore the warning.
